README.md

```diff
@@ -40,3 +40,113 @@ Run `helm upgrade --install <release-name> <path-to-chart> --namespace <namespac
 ### Default Namespace
 
 This installer is incompatible with `default` namespace installations, install and uninstall commands must include `--namespace <target-namespace>` or the context namespace must be changed, e.g. `oc project <target-namespace>`.
+
+## GitOps/ArgoCD Configuration
+
+As part of this section you will find information about the following configuration methods:
+
+1. Configuring GitOps/ArgoCD with the configuration script after using our `ai-rhdh-installer`
+2. Configuring GitOps/ArgoCD manually after using our `ai-rhdh-installer`
+3. Configuring GitOps/ArgoCD with the configuration script for a pre-existing ArgoCD and RHDH instance
+4. Configuring GitOps/ArgoCD manually for a pre-existing ArgoCD and RHDH instance
+
+### Prerequisites
+<!---
+TODO: Once RHDH configuration is complete we should link a reference to it here.
+-->
+- In order to allow the configuration to be completed you must first have a working RHDH instance (for example you are able to login and view developer hub).
```

Comment:
```suggestion
- In order to allow the configuration to be completed you must first have a working Red Hat Developer Hub instance (for example you are able to login and view Developer Hub).
```
small nit: if we haven't already, expand RHDH and capitalize

Reply:
After the first expansion of Red Hat Developer Hub, we can use the acronym RHDH

Reply:
My latest commit expands RHDH to Red Hat Developer Hub and outlines the acronym before referencing RHDH throughout the docs